spring-session/src/main/java/org/springframework/session/data/mongo/MongoSessionRepository.java

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Please rename to MongoOperationsSessionRepository to match existing naming conventions